basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,220 @@
 const helpers = require('./project-1');
 
 // start testing!
+it('should mult by ten', () => {
+    const expected = 100;
+
+    const actual = helpers.multiplyByTen(10);
+
+    expect(actual).toEqual(expected);
+});
+
+it('should sub by five', () => {
+    const expected = 10;
+
+    const actual = helpers.subtractFive(15);
+
+    expect(actual).toEqual(expected);
+});
+
+it('should compare string lgth', () => {
+    const expected = true;
+
+    const actual = helpers.areSameLength('node', 'smal');
+
+    expect(actual).toEqual(expected);
+});
+
+it('compare values for equality', () => {
```

Comment:
For each `it` describe what the test should do. For example, `it('should return true when two numbers are equal')`